### ISSUE TITLE
Preview /editting of ODF in Collabora no longer works in 2.22.6 (build 7) which works from build (3) and up

### ISSUE THREAD

**Issue by sebastiaanveld** [nextcloud/ios#723]
Expected behaviour                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
In App 2.22.6 (3) up to (5) when selecting an ODF document opened the document inline showing the document in Colabora which is setup behing my NextCloud server. After updating to 2.22.6 (7) this no longer works. This is mentioned in 2.2.6 (3) release note as "#715 good with Collaora .odt"

Would be great to restore this function again as IOS has no (real good) possibility to (pre)view and edit ODF documents. Th ebehavior in (3) to build (6) allowed viewing and editing of ODF documents in Collabora/LibreOffice Online.

Actual behaviour                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               
When now in App 2.22.6 (7) selecting an ODF document for preview nothing happens and just the NextCoud icon is shown on the screen.

Steps to reproduce                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             
1) Have a ODF file in a Documents folder of your NextCloud system. Have Collabora (latest version) CODE setup for editing documents.
2) Start the IOS App, logon and select an ODF document for (pre)view. In App 2.22.6 (3) up to (6) the document is the opened in CODE while after updating the App to 2.22.6 (7) this no longer happens and no preview is generated too. Just a black screen with NextCloud logo is shown.

iOS version
12.1 on iPhone X

App version
2.22.6 (7)

Server configuration
NextCloud 14.0.3, Latest Collabora Code, HAproxy in fornd.

Operating system**:
Linux/Tech and Me appliance

Web server:**
Apache

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)
NextCloud 14.0.3

**Comment by marinofaggiana** [comment]
@sebastiaanveld I have make a test (7) and this build works with collabora ... (open collabora for view/editing file) ... I don't understand where is the issue... black screen with NextCloud logo is shown ? mmmm strange ... do you have an account for me ?

**Comment by sebastiaanveld** [comment]
Can I drop you a mail with credentials?

**Comment by marinofaggiana** [comment]
yes, with inside a odf file

**Comment by sebastiaanveld** [comment]
Send to ios @ nextcloud dor com

**Comment by sebastiaanveld** [comment]
Mmm just updated to the Collabora 3.0.5 connector and found it may "work" again accessing the file with the IOS App; it just takes >60 sec's for a small doc to open meanwhile the App's ¨loader screen" (white background with Nextcloud logo) is shown. Then Collabora/LO shows up an the document opens. In the previous IOS App version just immediately the document opened in Collabora/LO.

**Comment by marinofaggiana** [comment]
@sebastiaanveld I close this issue, my account in your server is precious for the test :)

**Comment by sebastiaanveld** [comment]
Hi Marino,

Thx for your help! I fixed the issue I think last night. 

Regards Sebastiaan

Verstuurd vanaf mijn iPhone

**Comment by sebastiaanveld** [comment]
@marinofaggiana updating to **2.22.6 build 17** fined this issue. I just can now open documents with the app as fast as it worked up to builds 7.

**Comment by sebastiaanveld** [comment]
Think I found the issue; after upgrading to 14.0.4 the DB update failed resulting in it not starting anymore. Removed transactions in progress and remembered I ran into such also last time after a server reboot. Seems to do with uploades in progress on the App somehow I believe but this is hard to troubleshoot at this point. Just after starting the DB again I could with the App and opening docs in COOL/LOOL goes as fast again as before... that was a week ago. So fingers crosses;)
